connectors/redis/redis.go

```diff
@@ -40,6 +41,19 @@ type SimpleRedis interface {
 	Shutdown() error
 }
 
+// Scope is a namespace wrapper around a stats reporter, ensuring that
+// all emitted values have a given prefix or set of tags.
+type Scope interface {
+	// Counter returns the Counter object corresponding to the name.
+	Counter(name string) tally.Counter
```

Comment:
IMHO would shouldn't bring in the `tally` dependency here. It ties people to a very specific stats logging framework. This line is the only reason we need to import it, and so I'd say the `Counter` function shouldn't return a `tally.Counter`, but instead an interface that we define.

This will of course mean that we can't directly pass a tally counter to the constructor for this connector (bummer). We'll have to write a little, thin adapter so that a Tally scope works with a "dosa" scope.